components/grade-table.js

```diff
@@ -3,20 +3,20 @@ class GradeTable {
     this.tableElement = tableElement;
     this.noGradesElement = noGradesElement;
   }
-  updateGrades(grades) {
+  updateGrades(array) {
     this.noGradesElement.className = "d-none";
     this.destroy();
     var totalGrade = 0;
-    for(var i = 0; i < grades.length; i++) {
-      var rowElement = this.renderGradeRow(grades[i], this.deleteGrade, this.identityNum);
+    for(var i = 0; i < array.length; i++) {
+      var rowElement = this.renderGradeRow(array[i], this.deleteGrade, this.identityNum, i);
       this.tableElement.appendChild(rowElement);
-      totalGrade += grades[i].grade;
+      totalGrade += array[i].grade;
     }
-    if (grades.length === 0) {
-      this.noGradesElement.classList.remove = "d-none";
+    if (array.length === 0) {
+      this.noGradesElement.classList.remove("d-none");
       return 0;
     }
-    return totalGrade/grades.length;
+    return totalGrade/array.length;
   }
```

Comment:
Why the name change? Does `array` make it easier to understand what it is than `grades`?

Reply:
Yeah, the change made it more understandable to me because I thought line 13 looked confusing without the change.